src/scripts/setup-iasql-db.sh

```diff
@@ -8,36 +8,37 @@ echo "\nCreating an iasql db..."
 connectres=$(curl http://localhost:8088/v1/db/connect/iasql)
 export IASQL_USERNAME=$(jq -r '.user' <<<"$connectres")
 export IASQL_PASSWORD=$(jq -r '.password' <<<"$connectres")
+echo $connectres
```

Comment:
why is that needed?

Reply:
This was debug logging on my part. Going to remove.